.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# .vscode work-space file that contains settings.json for word checks. 
+.vscode
```

Comment:
Makes sense to keep project settings personal if everyone has their own.

However, Arsh added a `launch.json` file in the `.vscode` folder. Want to check in with him how you want to handle the vscode folder? 

If you want to share a debug launch.json you could explicitly add that file or be fancy and do this to ignore the folder _except_ that file.

```
.vscode
!.vscode/launch.json
```

Found this and the VS Code team themselves share stuff, but on a lot of projects I've been on we haven't so either way works
https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control